### DISCUSSION TITLE
Arrow.js + Storybook

### DISCUSSION THREAD

**Discussion by jonkwheeler** [standardagents/arrow-js#43] [General]
In case it helps someone, I was able to add Arrow.js to Storybook pretty easily.

Create a blank folder. 

Start with the HTML version of Storybook.

Run the following inside the folder and select HTML as the type you want to install.

```
npx storybook init
```

Then add Arrow.js.

```
npm install @arrow-js/core
```

Add a component.
```
/* PrimaryButton.js */

import { html } from "@arrow-js/core";

export const PrimaryButton = ({ text = "Hello World!" } = {}) => {
  return html`<button>${text}</button>`;
};

```

Add a story.
```
/* PrimaryButton.stories.js */

import { PrimaryButton } from "./PrimaryButton";

export default {
  title: "Sample",
};

const div = document.createElement("div");
div.id = "story";

export const Sample = () => {
  PrimaryButton()(div);

  return div;
};
```

This worked for me.

------

I made a repo just in case - https://github.com/jonkwheeler/storybook-arrowjs-boilerplate

**Comment by justin-schroeder** [comment]
Awesome work 👍
